tests/test_resources/metal_nxo_features.json

```diff
@@ -0,0 +1,132 @@
+{
```

Comment:
Not a strong preference, but I think when the JSON is records oriented, it can be a bit useful for manual inspection. Although this format is more compact. WDYT?

Reply:
I agree that's a good idea! I'll make the change.